lighttpd_rust_amalgamated/Cargo.toml

```diff
@@ -9,13 +9,9 @@ publish = false
 edition = "2021"
 autobins = false
 
-[lib]
-name = "lighttpd_rust_amalgamated"
-path = "lib.rs"
-crate-type = ["staticlib", "rlib"]
 [[bin]]
 path = "src/lighttpd_amalgamated.rs"
 name = "lighttpd_amalgamated"
```

Comment:
If we rename `lighttpd_amalgamated.rs` to `main.rs` we don't need this anymore.

Reply:
62b3eb0